scripts/build-pages/markdown-renderer/link.ts

```diff
@@ -1,9 +1,15 @@
+import { resolve } from 'url';
+
 const isInternal = /^\/docs\/.*/;
 
-export default (href: string, title: string, text: string) => {
+export default function(href: string, title: string, text: string) {
+  if (this.options.baseUrl && href.startsWith('../')) {
```

Comment:
Should we apply this to all relative links? (`./`, `../`)

Reply:
Good point. I've updated the check so it applies to everything except fragment-only URLs. The `url.resolve` method handles other relative and absolute URLs correctly.